app/controllers/application_controller.rb

```diff
@@ -2,4 +2,11 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
   before_action :authenticate_user!
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:accept_invitation, keys: [:name])
```

Comment:
I worry that it's easy to find out the _what_ of this line, but not the _why_ - might be tricky to change if you're looking at it and new to the project.

First of all - do _I_ understand the "why"? This change is so that when someone accepts an invitation, they can submit a `name` field to be saved on their new user?

I'm not sure on the best approach to make this more expressive, but I'd probably at least want to change the name of this method to something more specific - maybe `allow_name_parameter_for_invitations` if I've understood the reasons correctly.

Reply:
Went with `allow_devise_to_set_attributes`

Reply:
@kjdchapman Just had a quick chat about the naming of this and feel like `configure_devise_permitted_parameters` is a good name.

It describes what this does within the Rails / Devise domain terminology.

Let me know what you think.

Reply:
Ok propose we go with this for now and can discuss

Reply:
I think we are splitting hairs a bit here, everyone has a valid point. Emile's suggestion seems the middle ground between keeping it technically correct and a very descriptive plain english method name.